src/styles/blocks/container.scss

```diff
@@ -0,0 +1,23 @@
+.main {
+  display: grid;
+  grid-template-columns: repeat(var(--column-number), 200px);
+  --column-number: 1;
+  padding-top: 50px;
+  padding-bottom: 72px;
+  align-self: center;
+  row-gap: 46px;
+  column-gap: 48px;
+
+  @media (min-width: 488px) {
+    --column-number: 2;
+  }
+
+  @media (min-width: 768px) {
+    --column-number: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --column-number: 4;
+  }
+
```

Comment:
**flex layout**: there should always be 4 cards in a row (not 2, 3 or 5)

Reply:
- ❗️❗️❗️ **MANDATORY if you're familiar with the `grid`**
  - use `grid` instead of `flex`
  - implement the [Grid Layout instead](./grid.md)
  
  # Catalog using grid layout
Implement the layout using `grid`
- Gap between cards should be: horizontal - `48px`, vertical - 46px
- Blocks should start from the left
- Media queries:
  - There should be four columns starting at `1024px`
  - There should be three columns starting at `768px`
  - There should be two columns starting at `488px`
  - There should be one column below `488px`
  
  As we are familiar with the Grid, we have to work with the rules given for grid, as the rule you noted is given when you construct the catalog using flex instead of grid.